test/averageRating.spec.js

```diff
@@ -0,0 +1,188 @@
+import { describe, expect, test } from "@jest/globals";
+import { getAverageRating } from "../src/api";
+
+const mockId = 2;
+const mockImdb = "tt2953050";
+const mockImdbRating = 7.8;
+
+//Function to calculate average rating
+function mockMath(mockRating, mockImdbRating) {
+  if (mockRating.data.length >= 5) {
+    let sumOfAll = 0;
+    const mockData = mockRating.data;
+
+    mockData.forEach((int) => {
+      sumOfAll += int.attributes.rating;
+    });
+    const mockAnswer = sumOfAll / mockRating.data.length;
+    return mockAnswer;
+  } else {
+    return mockImdbRating;
+  }
+}
+
+describe("ratingFilterTest()", () => {
```

Comment:
The name of the test here, `"ratingFilterTest()"`, doesn't seem to reference anything in the code. Sometimes, when a test focuses on testing a particular function, the name of that function can also be the name of the test suite. Here, you are using a syntax that indicates that `ratingFilterTest` is the name of some function, but there is no function that has that name. That's confusing to me.